config/initializers/scheduler.rb

```diff
@@ -0,0 +1,8 @@
+require 'rufus-scheduler'
+require_relative '../../lib/tasks/orders'
+
+scheduler = Rufus::Scheduler.new
+order = OrdersTask.new
```

Comment:
Maybe it's better to:

* rename `order` to `order_task`
* move it to `every` section, as far as in the current implementation it may track only those orders which had status `waiting` on the start of the server